pkg/kubelet/cm/devicemanager/plugin/v1beta1/api.go

```diff
@@ -46,4 +46,7 @@ const (
 	errInvalidResourceName = "the ResourceName %q is invalid"
 	// errBadSocket is the error raised when the registry socket path is not absolute
 	errBadSocket = "bad socketPath, must be an absolute path:"
+	// maxMsgSize use 16MB as the default message size limit.
```

Comment:
the comment is kind of just explaining code. A better comment will be: 

```suggestion
	// maxMsgSize use 16MB as the message size limit for Device Plugin communication to support cases of XXX devices configured by the Device Plugin.
```

Reply:
Also add a small note on why this is a reasonable number of devices (how one can achieve it in real world scenario)